spec/shr_lab_panels_vs.txt

```diff
@@ -0,0 +1,106 @@
+LNC#788-0       "Erythrocyte distribution width [Ratio] by Automated count"
+LNC#736-9  	    "Lymphocytes/​100 leukocytes in Blood by Automated count"
+LNC#5905-5  	"Monocytes/​100 leukocytes in Blood by Automated count"
+LNC#713-8  	    "Eosinophils/​100 leukocytes in Blood by Automated count"
+LNC#706-2  	    "Basophils/​100 leukocytes in Blood by Automated count"
+LNC#751-8  	    "Neutrophils [#/​volume] in Blood by Automated count"
+LNC#731-0  	    "Lymphocytes [#/​volume] in Blood by Automated count"
+LNC#742-7  	    "Monocytes [#/​volume] in Blood by Automated count"
+LNC#711-2  	    "Eosinophils [#/​volume] in Blood by Automated count"
+LNC#704-7  	    "Basophils [#/​volume] in Blood by Automated count"
+LNC#764-1       "Band form neutrophils/100 leukocytes in Blood by Manual count"
+LNC#740-1       "Metamyelocytes/100 leukocytes in Blood by Manual count"
+LNC#749-2       "Myelocytes/100 leukocytes in Blood by Manual count"
+LNC#783-1       "Promyelocytes/100 leukocytes in Blood by Manual count"
+LNC#26523-1     "Promyelocytes [#/volume] in Blood"
+LNC#13046-8     "Variant lymphocytes/100 leukocytes in Blood"
+LNC#709-6       "Blasts/100 leukocytes in Blood by Manual count"
+LNC#30376-8     "Blasts [#/volume] in Blood"
+LNC#19048-8     "Nucleated erythrocytes/100 leukocytes [Ratio] in Blood"
+LNC#30392-5     "Nucleated erythrocytes [#/volume] in Blood"
+LNC#8251-1      "Service comment"
+LNC#776-5       "Platelet mean volume [Entitic volume] in Blood by Rees-Ecker"
+LNC#30446-9     "Myelocytes [#/volume] in Blood"
+LNC#30433-7     "Metamyelocytes [#/volume] in Blood"
+LNC#26507-4     "Band form neutrophils [#/volume] in Blood"
+LNC#71695-1     "Immature granulocytes/100 leukocytes in Blood by Automated count"
+LNC#53115-2     "Immature granulocytes [#/volume] in Blood by Automated count"
+LNC#58413-6     "Nucleated erythrocytes/100 leukocytes [Ratio] in Blood by Automated count"
+LNC#18314-5     "Morphology [Interpretation] in Blood Narrative"
+LNC#21000-5     "Erythrocyte distribution width [Entitic volume] by Automated count"
+LNC#32207-3     "Platelet distribution width [Entitic volume] in Blood by Automated count"
+LNC#32623-1     "Platelet mean volume [Entitic volume] in Blood by Automated count"
+LNC#35332-6  	"Band form neutrophils/​100 leukocytes in Blood by Automated count"
+LNC#19023-1  	"Granulocytes/​100 leukocytes in Blood by Automated count"
+LNC#42250-1  	"Variant lymphocytes/​100 leukocytes in Blood by Automated count"
+LNC#58409-4  	"Other cells/​100 leukocytes in Blood by Automated count"
+LNC#30229-9  	"Band form neutrophils [#/​volume] in Blood by Automated count"
+LNC#43743-4  	"Variant lymphocytes [#/​volume] in Blood by Automated count"
+LNC#51383-8  	"Leukocytes other [#/​volume] in Blood by Automated count"
+LNC#58443-3  	"Other cells [#/​volume] in Blood by Automated count"
+
+ValueSet:       CMPMustSupportPanelMembersVS
```

Comment:
Move the following from the optional list to must-support list.
LNC#33914-3  	"Glomerular filtration rate/​1.73 sq M.predicted [Volume Rate/​Area] in Serum or Plasma by Creatinine-based formula (MDRD)"
LNC#50044-7  	"Glomerular filtration rate/​1.73 sq M predicted among females [Volume Rate/​Area] in Serum, Plasma or Blood by Creatinine-based formula (MDRD)"
LNC#48642-3  	"Glomerular filtration rate/​1.73 sq M predicted among non-blacks [Volume Rate/​Area] in Serum, Plasma or Blood by Creatinine-based formula (MDRD)"
LNC#48643-1  	"Glomerular filtration rate/​1.73 sq M predicted among blacks [Volume Rate/​Area] in Serum, Plasma or Blood by Creatinine-based formula (MDRD)"
LNC#1968-7  	"Bilirubin.direct [Mass/​volume] in Serum or Plasma"

Reply:
done